City-of-Helsinki/helsinki-paatokset
# PP-129: Issue dropdown

To test:

- You either need to have full lists of agenda items and meetings OR you can modify paatokset_ahjo_get_handlings() to return a mock data array. Array items need to be arrays with keys `value` and `class`, such as `['value' => 123, 'class' => 'board]`
- After you have data in order, navigate to issue entity page. Verify that dropdown works as intended. Note that there is no event for selecting items as the page is to be loaded over ajax in another ticket
- Verify that dropdown is accessible
